<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolicySetting</class>
 <widget class="QWidget" name="PolicySetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_security">
     <property name="text">
      <string>SecurityLevel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_security"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_PolicyScope">
     <property name="text">
      <string>PolicyScope</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_PolicyScope"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_enforce">
     <property name="text">
      <string>Enforcement</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_enforce"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_AdminInfoUrl">
     <property name="text">
      <string>AdminInfoUrl</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_AdminInfoUrl"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_ExecutableTypes">
     <property name="text">
      <string>ExecutableTypes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QTextEdit" name="textEdit_exec"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_apply">
     <property name="text">
      <string>&amp;Apply</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton_exportfile">
     <property name="text">
      <string>&amp;Export on File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
